Adapter/Imagick.php

```diff
@@ -392,6 +392,11 @@ protected function openPng($file)
         // TODO: Implement openPng() method.
     }
 
+    protected function openWebp($file)
+    {
+        // TODO: Implement openWebp() method.
```

Comment:
Can't this be handle now?

Reply:
I didn't wat to stray from the rest of the code base so I did not implement this